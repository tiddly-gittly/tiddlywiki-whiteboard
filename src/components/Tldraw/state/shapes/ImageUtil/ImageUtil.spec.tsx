import { Image } from '..'

describe('Image shape', () => {
  it('Creates a shape', () => {
    expect(Image.create({ id: 'image' })).toMatchSnapshot('image')
  })
})
