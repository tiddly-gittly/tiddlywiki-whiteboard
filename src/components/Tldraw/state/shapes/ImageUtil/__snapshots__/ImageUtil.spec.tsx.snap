// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image shape Creates a shape: image 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "id": "image",
  "name": "Image",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": true,
    "scale": 1,
    "size": "small",
  },
  "type": "image",
}
`;
